.github/workflows/ci.yml

```diff
@@ -31,7 +31,6 @@ jobs:
           api-code: ['*/OpenTelemetry.Api*/**', '!**/*.md']
           api-packages: ['src/OpenTelemetry.Api*/**', '!**/*.md']
           instrumentation: ['*/OpenTelemetry.Instrumentation*/**', 'test/TestApp.AspNetCore/**', '!**/*.md']
-          instrumentation-packages: ['src/OpenTelemetry.Instrumentation*/**', '!**/*.md']
```

Comment:
Yes, it is strange way to execute https://github.com/open-telemetry/opentelemetry-dotnet/tree/3db49191410ec0b0cbe6f754510a4dc7c2a86dd3/test/OpenTelemetry.Instrumentation.W3cTraceContext.Tests

Reply:
Fine to leave it alone. I don't think the design for W3cTraceContext.Tests is really all that great. It is verifying/testing bits/logic inside the OpenTelemetry.Api project. Ideally it would be a test inside OpenTelemetry.Api.Tests and not use any instrumentation projects to do its work. Just haven't had the time/desire to do something about it 🤣